src/physical-and-chemical-data/data/inorganic-compounds.ts

```diff
@@ -55,4 +55,734 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
+      referenceTemperature: {
+        value: 4,
+        units: 'C',
+        error: 0,
+        greaterOrLess: 'exact',
+      },
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      acetone: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      CS2: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+    },
+  },
+
+  AluminumBromide2: {
+    name: 'Aluminum bromide',
+    formula: 'AlBr3.6H2O',
+    formulaWeight: 374.78,
+    color: { nature: Colors.Colorless, adjective: Adjectives.Empty },
+    crystallineForm: {
+      nature: [CrystallineForms.Crystalline, CrystallineForms.Deliquescent],
+      adjective: Adjectives.Empty,
+    },
+    refractiveIndex: NaN,
+    specificGravity: {
+      value: NaN,
+      substanceTemperature: {
+        value: 20,
+        units: 'C',
+        error: 5,
+        greaterOrLess: 'less',
+      },
+      reference: SpecificGravityReference.Water,
+      referenceTemperature: {
+        value: 0,
+        units: 'C',
+        error: 0,
+        greaterOrLess: 'exact',
+      },
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      hotWater: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+      CS2: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Empty,
+      },
+    },
+  },
+
+  AluminumCarbide: {
+    name: 'Aluminum carbidde',
+    formula: 'Al4C3',
+    formulaWeight: 143.96,
+    color: { nature: Colors.Yellow, adjective: Adjectives.Empty },
+    crystallineForm: {
+      nature: CrystallineForms.Hexagonal,
+      adjective: Adjectives.Empty,
+    },
+    refractiveIndex: 2.7,
+    specificGravity: {
+      value: 2.95,
+      substanceTemperature: {
+        value: 20,
+        units: 'C',
+        error: 5,
+        greaterOrLess: 'less',
+      },
+      reference: SpecificGravityReference.Water,
+      referenceTemperature: {
+        value: 0,
+        units: 'C',
+        error: 0,
+        greaterOrLess: 'exact',
+      },
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Decomposes,
+        adjective: Adjectives.Empty,
+        value: 'CH4',
+        temperature: {
+          value: NaN,
+          units: 'C',
+          error: 0,
+          greaterOrLess: 'less',
+        },
```

Comment:
Sir @maneetgoyal  was able to store `CH4` without modifying the interface with nature as `Decomposes` please have a look

Reply:
Are you referring to `decomposes to CH4`?

Reply:
Yeah sir

Reply:
I am making some changes incrementally at the comment. Try to study the commits in parallel. If you see any issue, let me know